vcs/git/git.go

```diff
@@ -348,27 +348,24 @@ func (r *Repo) ClearChanges() error {
 
 // TODO(benjamin-rood) Check git version to ensure expected cmd and behaviour
 // by any git command-line actions
+// Error codes:
+// 129 incorrect arguments supplied to git
 func gitCMD(args ...string) (out string, err error) {
 	cmd := exec.Command("git", args...)
 	b, err := cmd.CombinedOutput()
 	out = string(b)
-	return out, errors.Annotate(err, out)
 
-	// TODO(waigani) stdout is empty?
-	// cmd := exec.Command("git", args...)
-	// e := &bytes.Buffer{}
-	// o := &bytes.Buffer{}
-	// cmd.Stderr = e
-	// cmd.Stdout = o
-	// stderr = string(e.Bytes())
-	// stdout = string(o.Bytes())
-	// err = cmd.Run()
-	// if err != nil {
-	// 	gitargs := strings.Join(args, " ")
-	// 	return "", stderr, errors.Annotate(err, "git args: `"+gitargs+"` stdout: "+stdout+" stderr: "+stderr)
-
-	// }
-	// return strings.TrimSpace(stdout), stderr, nil
+	if strings.Contains(out, "unknown switch `4'") {
```

Comment:
Could you add a comment on this line explain this? It's not totally obvious why we'd do this.

> For users running older versions of git, run without the -4 flag.

^ should be fine.